editor/store/effects.js

```diff
@@ -530,4 +557,6 @@ export default {
 		window.fetch( window._wpMetaBoxUrl, fetchOptions )
 			.then( () => store.dispatch( metaBoxUpdatesSuccess() ) );
 	},
+
+	INSERT_BLOCKS: removeDefaultEmptyPrecedingBlock,
```

Comment:
I like that it's a separate documented function though it creates inconsistency between the effect handlers, we might want to move them all to separate functions.

Reply:
>I like that it's a separate documented function though it creates inconsistency between the effect handlers, we might want to move them all to separate functions.

Yeah, this was trying something new, since I didn't like the disorganization of this file, which is largely a result of a disconnect between the action types upon which they act, and the behaviors themselves. This had also encouraged including many different side effects in the same handler, when in-fact the value can be an array of functions, where each is an individual side effect for that particular action. Finally (and a more minor point), our unit tests should be testing the behavior of the side effect, not the original firing action.